src/com/dabomstew/pkrandom/newgui/Bundle.properties

```
@@ -635,4 +638,20 @@ GUI.tpEliteFourUniquePokemonCheckBox.text=Pokemon League Has Unique Pokemon:
 GUI.tpEliteFourUniquePokemonCheckBox.toolTipText=<html>Enabling this setting will ensure that Pokemon League Trainers (Elite Four, Champions and BW1 N/Ghetsis) <br />are given the specified number of unique Pokemon that will not appear on any other Trainer.<br />The Elite Four Trainers' highest level Pokemon are the ones that will be unique;<br/>in the case of a level tie, it will be the Pokemon further back in the party.<br />Does not apply to postgame Pokemon League battles.
 GUI.tpBetterMovesetsCheckBox.text=Better Movesets
 GUI.tpBetterMovesetsCheckBox.toolTipText=<html>Attempts to give Trainer Pokemon better movesets by including TM moves/tutor moves/egg moves/pre-evolution moves,<br />and picking moves that synergize with the Pokemon's ability/stats/other moves.
-
+BatchRandomizationSettingsDialog.title=Batch Randomization Settings
+BatchRandomizationSettingsDialog.enableCheckBox.text=Enable batch randomization
+BatchRandomizationSettingsDialog.enableCheckBox.toolTipText=If selected, generate the number of randomized ROMs specified.
+BatchRandomizationSettingsDialog.startingIndexLabel.text=Starting index
+BatchRandomizationSettingsDialog.startingIndexSpinner.toolTipText=Starting index for the first generated ROM.
+BatchRandomizationSettingsDialog.numberOfRandomizedROMsLabel.text=Number of randomized ROMs
+BatchRandomizationSettingsDialog.numberOfRandomizedROMsSpinner.toolTipText=Number of ROMs to generate in a single batch.
+BatchRandomizationSettingsDialog.outputDirectoryButton.text=Output directory...
+BatchRandomizationSettingsDialog.outputDirectoryButton.toolTipText=Choose the output directory for your randomized ROMs.
+BatchRandomizationSettingsDialog.fileNamePrefixLabel.text=File name prefix
+BatchRandomizationSettingsDialog.fileNamePrefixTextBox.toolTipText=File name prefix for the generated ROMs.
+BatchRandomizationSettingsDialog.autoAdvanceIndexCheckBox.text=Automatically advance index after batch
+BatchRandomizationSettingsDialog.autoAdvanceIndexCheckBox.toolTipText=If selected, the starting index will be automatically updated to the next index after a batch is ran.
```

Tiny grammar nitpick.
```suggestion
BatchRandomizationSettingsDialog.autoAdvanceIndexCheckBox.toolTipText=If selected, the starting index will be automatically updated to the next index after a batch is run.
```

---

Oops. Fixed.